django_stubs_ext/tests/test_monkeypatching.py

```diff
@@ -7,12 +7,15 @@
 from _pytest.monkeypatch import MonkeyPatch
 from django.db.models import Model
 from django.forms.models import ModelForm
+from django.views import View
 from typing_extensions import Protocol
 
 import django_stubs_ext
 from django_stubs_ext import patch
 from django_stubs_ext.patch import _need_generic, _VersionSpec
 
+extra_classes = [View]
```

Comment:
Let's create a separate `def test_` for this

Reply:
Done!